components/raftstore-v2/src/operation/command/admin/split.rs

```diff
@@ -704,15 +704,23 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         let region_id = self.region_id();
         if self.storage().has_dirty_data() {
             let tablet_index = self.storage().tablet_index();
-            let mailbox = store_ctx.router.mailbox(region_id).unwrap();
+            let mailbox = store_ctx.router.mailbox(region_id);
+            if mailbox.is_none() {
```

Comment:
what if it's not shutdown? This will miss a code bug. Maybe we should check tikv_util::thread_group::is_shutdown. If it's not shutdown, it should panic.

Reply:
Good catch. I have added the assertion.